phost/src/main.rs

```diff
@@ -360,12 +362,61 @@ async fn batch_sync_block(
     Ok(synced_blocks)
 }
 
+async fn sync_tx_to_chain(sequence: &mut u32, endpoint: String) -> Result<(), Error> {
+    let query = Query {
+        contract_id: 2,
+        nonce: 0,
+        request: ReqData::TransferToChain {sequence: *sequence},
+    };
+
+    let query_value = serde_json::to_value(&query)?;
+    let payload = Payload::Plain(query_value.to_string());
+    let query_payload = serde_json::to_string(&payload)?;
+    println!("query_payload:{}", query_payload);
+    let info = req_decode("query", QueryReq { query_payload: query_payload}).await?;
+    println!("info:{:}", info.plain);
+    let transfer_to_chain: TransferToChain = serde_json::from_str(&info.plain)?;
+    let transfer_queue = transfer_to_chain.transfer_to_chain.transfer_queue;
+    if transfer_queue.len() == 0 {
+        return Ok(());
+    }
+
+    let default_client = subxt::ClientBuilder::<subxt::DefaultNodeRuntime>::new()
```

Comment:
Why another client with `DefaultNodeRuntime`? Can we reuse the existing `PhalaRuntime` from the outside functions?